doc/source/reference/ufuncs.rst

```diff
@@ -327,6 +327,12 @@ advanced usage and will not typically be used.
     multiple outputs is deprecated, and will raise a warning in numpy 1.10,
     and an error in a future release.
 
+    If 'out' is missing or None, a uninitialized return array is created.
```

Comment:
How about `if 'out' is None (the default)`, rather than mentioning "missing"?

Reply:
adopted